presto-main/src/main/java/com/facebook/presto/util/MoreLists.java

```diff
@@ -29,5 +30,12 @@
                 .collect(toImmutableList());
     }
 
+    public static <T> List<T> filter(List<T> elements, Predicate<T> predicate)
```

Comment:
Does it matter that `elements` is a list? Can it be any `Collection`? Or `Iterable` (perhaps with the help of `Strems.stream(Iterable)`)?

I have also a problem whenever a see `filter(list, ...)` in the code. Is it filtered copy (your new method) or filtered view (`Iterables.filter`, the only candidate I knew previously).
Can the method be named `filteredCopy`? I know this is more verbose, but it's _not_ going to be used in `filteredCopy(transformedCopy(filteredCopy(...)))` cases, streams are for that.